SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSubviews.cpp

```diff
@@ -0,0 +1,216 @@
+#include "Interop/ActorSubviews.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/InitialOnlyFilter.h"
+#include "Schema/Restricted.h"
+#include "Schema/Tombstone.h"
+#include "Schema/UnrealMetadata.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/SubView.h"
+#include "SpatialView/ViewCoordinator.h"
+
+namespace SpatialGDK
+{
+namespace ActorSubviews
+{
+struct FMainActorSubviewSetup
+{
+	static bool IsActorEntity(const Worker_EntityId EntityId, const EntityViewElement& Entity, USpatialNetDriver& NetDriver)
```

Comment:
nit: It feels a bit inconsistent to me to have the implementation for this one inline, but the other ones below split between definition and declaration